cmd/pj-rehearse/main.go

```diff
@@ -265,37 +267,45 @@ func rehearseMain() int {
 	}
 	loggers := rehearse.Loggers{Job: logger, Debug: debugLogger.WithField(prowgithub.PrLogField, prNumber)}
 
-	toRehearse := diffs.GetChangedPresubmits(masterConfig.Prow, prConfig.Prow, logger)
-	metrics.RecordChangedPresubmits(toRehearse)
-	metrics.RecordOpportunity(toRehearse, "direct-change")
+	periodicsToRehearse := diffs.GetChangedPeriodics(masterConfig.Prow, prConfig.Prow, logger)
+	// TODO: metrics
```

Comment:
Why not here?

Reply:
Sure, I will include the metrics in this PR.